lua/cstm/buffer/code_action.lua

```diff
@@ -0,0 +1,10 @@
+local goimport = function()
```

Comment:
rename this `organize_imports` as this is something available in other LSP servers.